### ISSUE TITLE
E-pip is not written in Rust

### ISSUE THREAD

**Issue by Kyletsit** [Fitli/e-pip#1]
### Important Information
Provide following Information:

- e-pip version: latest from git

### Reproduction steps
In the e-pip source directory, run `find -name '*.py'`, followed by `find -name '*.rs'`.

### Expected behavior
The first command (`find -name '*.py'`) should produce no output. The second command (`find -name '*.rs'`) should print a list of e-pip's Rust source files.

### Actual behavior
The first command lists an alarming amount of `*.py` files. These are source code written in Python, an unsafe and slow programming language. The second command produces no output at all, which suggests that e-pip is, in fact, not written in Rust. Rust is a modern, safe programming language, featuring fearless abstractions and zero-cost concurrency. It's objectively superior to Python and every other language in every way, therefore e-pip should be rewritten in Rust as soon as possible.

### Log file
![Screenshot_20221213_171300](https://user-images.githubusercontent.com/67126208/207385428-ade65b03-8ba1-4799-a12c-8fba874fe978.png)

**Comment by Ladme** [comment]
I completely agree with you! The fact that the current version of e-pip is written in Python is extremely concerning. Rust is a fantastic choice for a programming language, with its emphasis on safety and performance. I completely endorse the idea that e-pip should be rewritten in Rust as soon as possible. The benefits it would bring to the program would be immense, and we would be able to have a much more reliable and efficient bot. Let's push for this change to happen!

**Comment by Kyletsit** [comment]
hi chatgpt
